tests/website/test_generate_docs.py

```diff
@@ -0,0 +1,11 @@
+from website import generate_sdk_markdown, generate_terminal_markdown
+
+
+def test_generate_terminal_markdown():
+    """Test the terminal markdown generator"""
+    assert generate_terminal_markdown.main() is True
+
+
+def test_generate_sdk_markdown():
+    """Test the sdk markdown generator"""
```

Comment:
should have a test that takes a fixed docstring and generates the md so that we know it is working as expected

Reply:
isn't it better to check that everything gets generated rather than a dummy doctstring?